### ISSUE TITLE
Comparison

### ISSUE THREAD

**Issue by arschmitz** [axemclion/perfjankie#19]
Currently i'm using this to research different existing approaches to the markup and css of different components as preparation to creating a new css framework. It would be really nice if there was a comparison view where I could compare the results of different tests. perhaps a table + graph with all of the selected pages plotted?

**Comment by axemclion** [comment]
Do you have a UI in mind for this? Are you looking at comparing different "runs", or different "pages".

**Comment by arschmitz** [comment]
Im looking for comparison of different "pages" for example http://104.236.81.132:5984/css-perf/_design/site/index.html#/page-select i would like to be able to compare these 4 pages. Each is testing the button of a different framework so a comparison would be useful to evaluate the approaches of the different frameworks. 

as far as a ui using frame rate as an example the graph would plot look something like it does now with the lines except there would be one for each page and a legend. Below this you could have a table with the numbers for each framework. For the overview page you could do essentially the same adjusting where appropriate for different chart types. 

Also when i showed this to collaborators this afternoon this was also their first request.

**Comment by axemclion** [comment]
When I started working on perfjankie, I wanted to replicate http://bench.topcoat.io/. Thus, the idea was not to compare different pages, but look at the perf trends of a single page (or scenario) as various commits change code to that page. Hence the line charts. 
I could see that you could make the buttoms from different framework look like different version of the same page, but I understand that the line chart is not the best way to compare things. 
I think we would need to see how we can continue to support trends in the same page, while adding the ability to compare different pages.

**Comment by arschmitz** [comment]
@axemclion I don't think you actually need to do anything special to support both. I was thinking it would work like a typical product comparison on shopping sites. checkboxes select some number of pages and hit a comparison button. This would take you to something similar to the overview page that exists now just with overlays for graphs / tables for numbers. Line graphs I think would still work fine also.

**Comment by axemclion** [comment]
Yeah - this sounds like a workable idea.

**Comment by arschmitz** [comment]
Yeah I didn't want to change the current views at all they are very useful for CI, ( which I also plan on using this for, its just not the current use I have ). I only wanted to add a new one with the comparison.

**Comment by axemclion** [comment]
That sounds good. Would be great if you could do a PR for this feature - will be glad to add it in.

**Comment by prasunanand** [comment]
I think I can create the graph. Recently I worked on this http://stats.biojs.net/
